### ISSUE TITLE
enhancement: a11y rule "no-noninteractive-tabindex" fails with a warning. 

### ISSUE THREAD

**Issue by FatumaA** [open-sauced/app#4135]
### Describe the bug

[jsx-a11y eslint plugin](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) was recently added to the project and this rule - jsx-a11y/no-noninteractive-tabindex - fails with a warning.

This issue offers more context: https://github.com/open-sauced/app/issues/2264

### Steps to reproduce

The following components and files are causing the rule to fail:

1. components/atoms/Card/card.tsx

**Comment by rothardo** [comment]
is this up for grabs? @nickytonline

**Comment by FatumaA** [comment]
@rothardo yes, to claim comment ".take" on the issue

**Comment by rothardo** [comment]
.take

**Comment by FatumaA** [comment]
Oops let me remove my assignment

**Comment by rothardo** [comment]
.take
